### ISSUE TITLE
Support for View Components

### ISSUE THREAD

**Issue by artiomchi** [T4MVC/R4MVC#44]
ASP.NET Core MVC added a new entity called View Components. They're similar to partial views / child actions, but more powerful and much better implemented.

I think R4MVC should also add support for them (although the initial core functionality takes priority ofc)

docs: https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-components

**Comment by kevinkuszyk** [comment]
Agreed.  Ok if I move this one to the backlog?

**Comment by artiomchi** [comment]
Of course, I'd rather get the basic functionality out first, and then start adding extras
